scout.py

```diff
@@ -5,4 +5,11 @@
 from ScoutSuite.__main__ import run_from_cli
 
 if __name__ == "__main__":
+    # sys.argv = ['scout.py', 'azure', '--cli', '--force']
+    sys.argv = ['scout.py', 'azure', '--service-principal', '--force','--tenant',
+                '0cc90829-0d8e-40d6-ba9c-aea092ba7de5']
+
+    # sys.argv = ['scout.py', 'azure', '--user-account-browser', '--force', '--tenant',
+    #             '0cc90829-0d8e-40d6-ba9c-aea092ba7de5']
+
```

Comment:
```suggestion
```

Reply:
Best to configure these in your IDE run config.

Reply:
Thanks! didn't know about that. Great idea